### PR TITLE
Develop adminsearch staticindex

### PR DESCRIPTION
**Pull request by paulklinkenberg**
As discussed in the railo-team list: the search functionality in the Railo-admin currently does not work anymore, and should use pre-compiled index files. The changes in this pull request make sure there is no local indexing being done anymore, and the admin.search.index.cfm file can now be used in the railo-builder to create the searchindex files.

### CODE REVIEW

**Comment by micstriit** [comment]
please have in mind that the develop branch is for Railo 5.0, not 4.2

**Comment by paulklinkenberg** [comment]
Oow, that sucks. Let me try again then :)
